.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length=88
+extend_ignore=W503,E203
```

Comment:
I think the comment should be left here. Cause nobody knows what any of these are.

Reply:
Skill issue :trollface:

I'll add in comments lol